.github/workflows/publish-docker-containers.yml

```diff
@@ -114,13 +119,16 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           context: .
-          push: ${{ inputs.publish || startsWith(github.ref, 'refs/tags/') }}
-          file: dockerfiles/${{ matrix.app =='trading' && 'Dockerfile.next' || 'Dockerfile.cra' }}.dist
-          build-args: APP=${{ matrix.app }}
-          platforms: ${{ inputs.archs || 'linux/amd64, linux/arm64' }}
+          push: true
+          # push: ${{ inputs.publish || startsWith(github.ref, 'refs/tags/') }}
+          file: dockerfiles/Dockerfile.cra.dist
+          build-args: |
+            APP=${{ matrix.app }}
+            NODE_VERSION=${{ steps.tags.outputs.npmVersion }}
+          # platforms: ${{ inputs.archs || 'linux/amd64, linux/arm64' }}
           tags: |
-            vegaprotocol/${{ matrix.app }}:latest
-            vegaprotocol/${{ matrix.app }}:${{ steps.tags.outputs.version }}
+            ghcr.io/vegaprotocol/frontend/${{ matrix.app }}:latest
```

Comment:
Why we are not publishing to dockerhub anymore?

Reply:
we will be using this just internally at the moment -> so ghcr